PWGGA/GammaConv/AliAnalysisTRDEfficiency.h

```diff
@@ -94,26 +54,22 @@ class AliAnalysisTRDEfficiency : public AliAnalysisTaskSE
         THnSparse*              fhgevent8;
         THnSparse*              fhgevent9;
         
-        //event counter
-        TH1F*                   fhgevent;    // event counter
-        TH1F*                   fhevent;
-        
-        //track the events
-        THnSparse*              fhtrckvnt;   // track event
-        THnSparse*              fhtrckvnthqu;// track hqu event
-        TH1*                    fhtrvnt;
-        TH1*                    fhtrvnthqu;
-        TList*                  lsttrckvnt;
-        TList*                  lsttrckvnthqu;
-        //AliConversionPhotonCuts fConversionCuts;
-        
-        TH2F*                   fhgetaphi1;
-        TH1F*                   fhgR1;
-        TH1F*                   fhgpt1;
-        TH2F*                   fhgetaphi5;
-        TH2F*                   fhgetaphi8;
-        TH2F*                   fhgetaphi9;
+        THnSparse*              fhg;
+        THnSparse*              fhpi0;
+        THnSparse*              fhpi0bkg;
+        THnSparse*              fhevent;
+        TH1F*                   fhev;
         
+        THnSparse*              fhgdghtr;
+        THnSparse*              fhgtest;
+
+        Int_t                   eventNumber = 0;
+        TObjArray               *flst;    
+        AliPIDResponse*         fPIDResponse;
+        AliTRDonlineTrackMatching* online;
+        //AliAnalysisTaskESDfilter*  esdfilter;
+        AliConversionSelection* convsel;
+
```

Comment:
the ClassDef should be incremented

Reply:
I'm not entirely sure what the ClassDef is defining so I don't know what it should be incremented to.

Reply:
Whenever you add or remove a member variable which is not transient, you should increment the ClassDef by 1

Reply:
I see, well it should be fixed now.

Reply:
using //! for fPIDReponse, fOnline and fConvsel should also be fine